tests/openssl/tests_3.unsupported

```diff
@@ -7,8 +7,6 @@
 ../../3rdparty/openssl/openssl/test/asn1_time_test.c
 # The MS CryptoAPI engine (capieng) is disabled.
 ../../3rdparty/openssl/openssl/test/cmsapitest.c
-# PSK is disabled by OE.
-../../3rdparty/openssl/openssl/test/dtls_mtu_test.c
```

Comment:
Has this test been removed?

Reply:
Thank you for the catch, I think this change might be a mistake, I'll double check

Reply:
Also fixing issue where these paths refer to OpenSSL 1 repo instead of OpenSSL 3.0